pennylane/tape/qscript.py

```diff
@@ -871,10 +879,15 @@ def copy(self, copy_operations=False):
         new_qscript._obs_sharing_wires_id = self._obs_sharing_wires_id
         return new_qscript
 
-    def __copy__(self):
+    def __copy__(self) -> "QuantumScript":
         return self.copy(copy_operations=True)
 
-    def expand(self, depth=1, stop_at=None, expand_measurements=False):
+    def expand(
+        self,
+        depth: int = 1,
+        stop_at: Optional[Callable[[Operation], bool]] = None,
```

Comment:
```suggestion
        stop_at: Optional[Callable[[Union[Operation, MeasurementProcess]], bool]] = None,
```

It's also called on measurement processes.  This has caused problems in the past, as a `stop_at` function wouldn't work with measurements, and then would cause errors.